bioblend/galaxy/datasets/__init__.py

```diff
@@ -125,7 +126,9 @@ def download_dataset(self, dataset_id, file_path=None, use_default_filename=True
                 file_local_path = file_path
 
             with open(file_local_path, 'wb') as fp:
-                fp.write(r.content)
+                for chunk in r.iter_content(chunk_size=1024):
```

Comment:
Can you please use `bioblend.CHUNK_SIZE` instead of the hardcoded 1024?

You will also need to add `import bioblend` at line 14.